azure-pipeline-templates/release-distro-tests.yml

```diff
@@ -130,6 +130,56 @@ steps:
       blobfuse2 unmount ${{ parameters.mount_dir }}
     displayName: 'Unmount ADLS'
 
+  # - script: |
```

Comment:
Why is this section commented out. If its dead code then we shall remove it from here.

Reply:
I'll bring it back in a future PR ni a few weeks. I didnt want her to get too deep into the pipeline code